src/calculateRentalCost.js

```diff
@@ -4,7 +4,18 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dayCost = 40;
```

Comment:
Consider using a constant for 'dayCost' to prevent unintentional reassignment and to better convey the intent that this value should not change.